apps/src/sites/studio/pages/home/_homepage.js

```diff
@@ -42,6 +45,7 @@ function showHomepage() {
   store.dispatch(initializeHiddenScripts(homepageData.hiddenScripts));
   store.dispatch(setPageType(pageTypes.homepage));
   store.dispatch(setLocaleEnglishName(homepageData.locale));
+  store.dispatch(setCurrentUserId(homepageData.currentUserId));
```

Comment:
I'm surprised we don't already have this information on this page!

Reply:
I was also surprised!!